lib/pengine/complex.c

```diff
@@ -1105,6 +1101,9 @@ common_free(pcmk_resource_t * rsc)
     g_list_free(rsc->private->this_with_colocations);
     g_list_free(rsc->private->location_constraints);
     g_list_free(rsc->private->ticket_constraints);
+    if (rsc->private->probed_nodes != NULL) {
```

Comment:
common_free() is the one place where the resource tables do have to be checked, since it can be called for unpack errors before the tables are created

Reply:
ah yes